### ISSUE TITLE
Cannot mount service bindings

### ISSUE THREAD

**Issue by connorb08** [cloudflare/miniflare#538]
I am using Miniflare in part of the workflow for a web app built on workers. It was working properly, until I added another worker and a service binding to that worker. When I try to use miniflare for development and testing, it throws an error that the service isn't mounted. I have the service mounted as shown in the docs, but it is throwing an error.

How it is mounted for API:
```
const mf = new Miniflare({
        mounts: {
          api: {
            rootPath: '../../api',
            wranglerConfigPath: true,
            packagePath: true
          },
        },
        scriptPath: packageJson.module,
        modules: true,
        wranglerConfigPath: true,
        port,
      });
```

I have a second mount for cli:
```
[miniflare.mounts]
api = "../api"
```

These are both pointed to the right place.

The error it is throwing is "Unexpected token 'export'". This doesn't really make sense to me, because it is valid code for the worker. 

```
[mf:err] MiniflareCoreError [ERR_MOUNT]: Error mounting "api"
    at Miniflare.#init (/host_path/node_modules/@miniflare/core/src/index.ts:612:19)
    at Miniflare.getPlugins (/host_path/node_modules/@miniflare/core/src/index.ts:1033:5)
    at createServer (/host_path/node_modules/@miniflare/http-server/src/index.ts:362:19)
    at startServer (/host_path/node_modules/@miniflare/http-server/src/index.ts:469:18)
    at main (/host_path/node_modules/miniflare/src/cli.ts:111:7)
[mf:err] Cause: /mounted_path/src/index.ts:11
export interface Env {
^^^^^^

SyntaxError: Unexpected token 'export'
```

The file is nothing crazy, it works perfectly fine on CF. 

```
export interface Env {

}

export default {
	async fetch(
		request: Request,
		env: Env,
		ctx: ExecutionContext
	): Promise<Response> {
		return new Response(JSON.stringify({message: "Site is connected to the API Worker!"}));
	},
};
```

I don't really need to use the other worker for the testing or development, so an option to ignore this would be acceptable as well, but this is causing issues with my development workflow and automated testing.

Thank you for any help!

**Comment by mrbbot** [comment]
Hey! 👋 It looks like your mounted Worker is using the modules format. You'll need to tell Miniflare this is using `modules` as it doesn't inherit this option from the parent (you can have multiple workers using a mix of modules/service-worker format). This should be a case of adding `modules: true` in your `api` object:

```diff
 const mf = new Miniflare({
   mounts: {
     api: {
       rootPath: '../../api',
       wranglerConfigPath: true,
       packagePath: true
+      modules: true
     },
   },
   scriptPath: packageJson.module,
   modules: true,
   wranglerConfigPath: true,
   port,
 })
```

**Comment by connorb08** [comment]
Thank you for the comment! I was able to fix the issue doing the following:

I set modules to true but it was still having an issue mounting it correctly. I went to the other worker and the wrangler.toml + package.json files there didn't have the module configuration set up correctly, and that was the issue. Wrangler was able to tell that it was a module, so I thought miniflare would be able to as well. When I tried to mount it as a module, it was going to the api directory and use the defaults there, where the module info wasn't configured correctly.
